linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_min_life_existing/bash/shared.sh

```diff
@@ -0,0 +1,16 @@
+# platform = Red Hat Virtualization 4,multi_platform_fedora,multi_platform_ol,multi_platform_rhel
+. /usr/share/scap-security-guide/remediation_functions
+{{{ bash_instantiate_variables("var_accounts_minimum_age_login_defs") }}}
+
+# only remediate interactive user accounts.
+for acct in $(awk -F: '$3 >= 1000 && 
+                       $3 != 65534 && 
+                       $3 != 65535 &&
+                       $3 != 4294967294 && 
+                       $3 != 4294967295 { print $1 }' /etc/passwd); do
+
+    # only remediate accounts that are actually failed.
+    if [ $(chage -l $acct | awk -F: '/Minimum/ {print $2}') -gt $var_accounts_minimum_age_login_defs ]; then
```

Comment:
Shouldn't the conditional  be `-lt`?

When the account has Minimum age `0` or `-1`, they should be set to the minimum age defined by the variable.
See also comments in the following test scenarios:
- accounts_password_set_min_life_existing/tests/incorrect_stig.fail.sh 
- accounts_password_set_min_life_existing/tests/incorrect_stig_blank.fail.sh

Reply:
Fixed.